install-deps.sh

```diff
@@ -65,7 +65,7 @@ function munge_debian_control {
 	sed -i -e 's/^# Crimson[[:space:]]//g' $control
     fi
     if $with_jaeger; then
-	sed -i -e 's/^# Jaeger[[:space:]]//g' $control
+	sed -i -e 's/^# Jaeger[[:space:]]\+//g' $control
```

Comment:
please update the title of the commit message. current it reads:

> debian: fix Built-Using parse error with Jaeger

Reply:
@tchaikov Please suggest me a better commit title if it's not good!

Reply:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes. probably you could follow the suggestion i made for another commit in this PR.

Reply:
@tchaikov I've seen a commits history for `install-deps.sh` file and it's starting with `install-deps:` https://github.com/ceph/ceph/commits/master/install-deps.sh
Should I use the `build` prefix or...?

Reply:
i'd suggest use `isntall-deps.sh`. it's much more specific than "build".

Reply:
@tchaikov I used the same `install-deps.sh` prefix in the commit title!

Reply:
thanks. it's much better now.